online_repos-main-repo-non-oss-disabled-20240109.json

```diff
@@ -0,0 +1,15 @@
+{
+  "area": [
+    {
+      "xpos": 26,
+      "ypos": 129,
+      "width": 225,
+      "height": 15,
+      "type": "match"
+    }
+  ],
+  "properties": [],
+  "tags": [
+    "main-repo-non-oss-disabled"

```

Comment:
Do you have a verification run?

Reply:
https://aquarius.suse.cz/tests/19984#step/online_repos/12

Reply:
Leap was not considered. But apparently havent been used there anyway so far